src/components/editor/EditorMainIssueController.jsx

```diff
@@ -65,6 +65,12 @@ export default class EditorMainIssueController extends FalcorController {
                 return false;
               }
             }
+            if (/<a.*?href\s*=\s*["'][^http].*?>/.test(article.html)) {
```

Comment:
Yep this is good as we discussed. Since you'll be changing things anyway I'm thinking it might be cleaner to add the `?` behind the two `\s*` though since it's only checking for whitespace it shouldn't be a problem as the `>` isn't included so it wouldn't be able to carry over to a new tag.

Also, I know that was my own suggestion, but though it is very unlikely we will ever see a tag called <ahref> in HTML6, especially when it is followed by an equals etc. etc., it'd be better form to do `<a\s.*?`.